lib/rbUtil.js

```diff
@@ -199,4 +180,11 @@ rbUtil.parseETag = function(etag) {
     }
 };
 
+// Store titles as MediaWiki db keys
```

Comment:
Adding MediaWiki-specific utilities in rbUtil feels a bit dirty. Having a mediawikiUtils.js file in mods might be a slightly cleaner alternative, although it's not really a module either.